src/style.css

```diff
@@ -1,100 +1,102 @@
 * {
   margin: 0;
   padding: 0;
-  box-sizing: border-box;
   list-style: none;
-  background: rgb(168, 161, 161);
-  background: white;
+  box-sizing: border-box;
 }
 
 body {
-  background: white;
-  min-height: 70vh;
-  width: 28em;
-  margin: 50px auto;
-  margin-top: 7em;
-  box-shadow: 0 3px 10px rgba(0, 0, 0, 0.664);
-}
-
-header {
-  margin: 1em auto 0;
-  padding: 1em;
+  display: grid;
+  place-items: center;
+  background-color: lightgrey;
+  min-height: 100vh;
+  min-width: 100vw;
 }
 
-.header {
-  display: flex;
-  justify-content: space-between;
-  width: 26em;
+.container {
+  background: white;
+  max-width: 2000px;
+  box-shadow: 0 3px 10px rgba(0, 0, 0, 0.664);
 }
 
-p {
-  font-size: 20px;
-}
-
-header,
-.form {
-  display: flex;
-  align-items: center;
-  justify-content: center;
+h1 {
+  color: #2555ed;
+  text-align: center;
+  margin: 1rem 5rem;
 }
 
 button {
   border: none;
+  background: white;
 }
 
-.form input,
-.form button {
+form input,
+form button {
   padding: 1rem;
-  font-size: 22px;
+  font-size: 20px;
   border: none;
   border-top: 1px solid rgb(197, 196, 196);
   border-bottom: 1px solid rgb(187, 187, 187);
   background: rgb(247, 245, 245);
   outline: none;
+}
 
-  /* width: 100%; */
```

Comment:
- [ ] Please remove commented code, it makes your code look messy so kindly remove it to make your code clean